daemon/daemon.go

```diff
@@ -404,6 +404,10 @@ func (daemon *Daemon) restore() error {
 							c.Unlock()
 						}
 					}
+				} else if !c.IsPaused() && alive {
+					daemon.updateHealthMonitor(c)
+					logrus.WithField("container", c.ID).
+						Info("Restored health monitor")
```

Comment:
This can use the `log` that's initialised above; https://github.com/moby/moby/blob/d5612a0ef8aa5c43f8345302e8d20b5fd25517b3/daemon/daemon.go#L324

```suggestion
					log.Info("restored health monitor")
```

Reply:
Indeed there is no reason to initialize a new instance of logrus, it is fixed. Thank you.